docs/development-guide/project-structure/components/accumulo-store.md

```diff
@@ -194,7 +194,6 @@ Ensure that when running an Accumulo instance, the user specified by the `accumu
 | vis1              | [VisibilityIT](https://github.com/gchq/Gaffer/blob/develop/integration-test/src/test/java/uk/gov/gchq/gaffer/integration/impl/VisibilityIT.java) |
 | vis2              | [VisibilityIT](https://github.com/gchq/Gaffer/blob/develop/integration-test/src/test/java/uk/gov/gchq/gaffer/integration/impl/VisibilityIT.java) |
 | public            | [SchemaHidingIT](https://github.com/gchq/Gaffer/blob/develop/core/graph/src/test/java/uk/gov/gchq/gaffer/integration/graph/SchemaHidingIT.java) |
-| private           | [ParameterizedLoaderIT](https://github.com/gchq/Gaffer/blob/develop/integration-test/src/test/java/uk/gov/gchq/gaffer/integration/impl/loader/ParameterizedLoaderIT.java#L63)
```

Comment:
This shouldn't be removed because the link is wrong. I think that test got refactored, this is probably okay: https://github.com/gchq/Gaffer/blob/develop/integration-test/src/test/java/uk/gov/gchq/gaffer/integration/impl/loader/UserLoaderIT.java

Reply:
No problem, couldn't find the ParameterizedLoaderIT file and presumed this had been deleted from the repo.